src/pages/store/item/[uuid].tsx

```diff
@@ -0,0 +1,88 @@
+import { Navbar } from '@/components/store';
+import AddCartButton from '@/components/store/AddCartButton';
+import ItemHeader from '@/components/store/ItemHeader';
+import SizeSelector from '@/components/store/SizeSelector';
+import { StoreAPI } from '@/lib/api';
+import config from '@/lib/config';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PrivateProfile, PublicMerchItemWithPurchaseLimits } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import { getDefaultMerchItemPhoto } from '@/lib/utils';
+import styles from '@/styles/pages/store/item.module.scss';
+import { GetServerSideProps } from 'next';
+import Image from 'next/image';
+import { useState } from 'react';
+
+interface ItemPageProps {
+  user: PrivateProfile;
+
+  item: PublicMerchItemWithPurchaseLimits;
+}
+const StoreItemPage = ({ user: { credits }, item }: ItemPageProps) => {
+  const [size, setSize] = useState<string | undefined>(item.options.length <= 1 ? 'Y' : undefined);
+  const [inCart, setInCart] = useState<boolean>(false);
+
+  const currOption =
+    item.options.length <= 1
+      ? item.options[0]
+      : item.options[item.options.findIndex(val => val.metadata?.value === size)];
+
+  return (
+    <div className={styles.navbarBodyDiv}>
+      <Navbar balance={credits} showBack />
+      <div className={styles.rowContainer}>
+        <Image
+          src={getDefaultMerchItemPhoto(item) || ''}
```

Comment:
I don't think `getDefaultMerchItemPhoto` returns undefined anymore, so you can get rid of the `|| ''`

```suggestion
          src={getDefaultMerchItemPhoto(item)}
```